### ISSUE TITLE
Regression with Norse

### ISSUE THREAD

**Issue by Eiad21** [norse/norse#254]
Is there a way to perform regression tasks with Norse? If so can you kindly provide an example notebook or a link from the documentation.

**Comment by cpehle** [comment]
Yes, it is possible to implement regression tasks, this is mostly down to the choice of Loss function. One thing that is tricky and an active area of research is how to best define a loss which only operates on spike times in order to do fit for example the behaviour of a Neuron model to measurements. I can look into adding an example in the documentation.

**Comment by ahenkes1** [comment]
One way could be to add a linear output layer, which converts a (sufficiently large) number of spikes to numeric values. This wouldn't be "pure", but it should work with standard losses. But I am curious to see other approaches. Looking forward to the examples, maybe with hints to publications?
Best

**Comment by cpehle** [comment]
I have some example code lying around that does regression on sum of sines, which I could add.
